src/utility/getClassesForSizingAndPositioning.test.js

```diff
@@ -0,0 +1,90 @@
+import {
+  getClassForAddedPackaging,
+  getClassForListedPackaging,
+  getClassForWeekdayCheckboxAlignment,
+} from "./getClassesForSizingAndPositioning";
+
+// Added products
+
+test("gives back the name of the class an image should have depending on which packaging it has", () => {
```

Comment:
I don't think it is important for what kind of element the class is used, just the class is important